src/providers/slot/slot.ts

```diff
@@ -81,13 +81,8 @@ export class SlotProvider {
     // if it's not Friday
```

Comment:
If this PR is accepted without issue, I will need to refactor the comment of this function.